apps/bfd-server/bfd-server-war/src/test/resources/openapi/headers.yaml

```diff
@@ -0,0 +1,162 @@
+      name: BlueButton-User
+      required: false
+      schema:
+        type: string
+      description: | 
+        The human-readable login ID (e.g. email address) in the frontend system for the user (e.g. beneficiary) 
+        that the data is being requested on behalf of.
+    BlueButton-BeneficiaryId-header:
+      in: header
+      name: BlueButton-BeneficiaryId
+      required: false
+      schema:
+        type: string
+      description: |
+        The unique beneficiary ID (e.g. Patient.id) from BFD for the user the beneficiary 
+        whose data is being requested.
+    x-forwarded-for-header:
+      in: header
+      name: x-forwarded-for
+      required: false
+      schema:
+        type: string
+      description: |
+        A standard header for identifying the originating IP address of a client connecting to 
+        a web server through an HTTP proxy or a load balancer.
+    BULK-CLIENTID-header:
+      in: header
+      name: BULK-CLIENTID
+      required: false
+      schema:
+        type: string
+      description: |
+        A unique identifier of the bulk client for whom this request is for (i.e. UUID, ACO ID, NPI, Part D Contract)
+    BULK-CLIENTNAME-header:
+      in: header
+      name: BULK-CLIENTNAME
+      required: false
+      schema:
+        type: string
+      description: A human readable name for the bulk client for whom this request is for to aid in tracing and debugging.
+    BULK-JOBID-header:
+      in: header
+      name: BULK-JOBID
+      required: false
+      schema:
+        type: string
+      description:  A unique identifier for the job (Job ID, UUID, etc.)
+    IncludeIdentifiers-header:
+      in: header
+      name: IncludeIdentifiers
+      required: false
+      schema:
+        type: string
+        enum: ["false", "true", "hicn", "mbi"]
+        default: "false"
+      description:  |
+        Do not set this header more than once; an arbitrary value will be selected if that happens. When set to mbi, 
+        BFD will include all of the known MBIs for the requested beneficiary (unhashed). When set to hicn, BFD will 
+        include all of the known HICNs for the requested beneficiary (unhashed). When set to true, BFD will include 
+        all of the known MBIs and HICNs for the requested beneficiary (unhashed). When set to false, BFD will not 
+        include any (unhashed) MBIs or HICNs for the requested beneficiary.
+    IncludeAddressFields-header:
+      in: header
+      name: IncludeAddressFields
+      required: false
+      schema:
+        type: string
+        enum: ["false", "true"]
+        default: "false"
+      description:  |
+        When set to true, BFD will include all of the detailed address data for the requested beneficiary. When set to 
+        false, BFD will not include detailed address data for the requested beneficiary. Please note that, even when 
+        false county and ZIP/postal codes will still be included for the specified beneficiary.
```

Comment:
As far as I can see in the code (BeneficiaryTranformerV2:172) we actually return state and postalCode when this is false, not county.

Reply:
Corrected.